Main.cc

```diff
@@ -440,7 +452,7 @@ void OnMessage(uWS::WebSocket<uWS::SERVER>* ws, char* raw_message, size_t length
             }
         }
     } else if (op_code == uWS::OpCode::TEXT) {
-        if (strncmp(raw_message, "PING", 4) == 0) {
+        if (sv_message == "PING") {
             ws->send("PONG");
```

Comment:
with the new version, it seems we need to explicitly specify a text output as well:

`ws->send("PONG", uWS::OpCode::TEXT);`